arangod/Scheduler/SupervisedScheduler.cpp

```diff
@@ -452,7 +461,8 @@ std::string SupervisedScheduler::infoStatus() const {
 
   return "scheduler threads " + std::to_string(numWorker) + " (" +
          std::to_string(_numIdleWorker) + "<" + std::to_string(_maxNumWorker) +
-         ") queued " + std::to_string(queueLength);
+         ") queued " + std::to_string(queueLength) +
+         "directly exec " + std::to_string(_jobsDirectExec.load(std::memory_order_relaxed));
```

Comment:
```suggestion
         " directly exec " + std::to_string(_jobsDirectExec.load(std::memory_order_relaxed));
```
let's add a whitespace for better readability